lib/grizzly/firmware_updates/firmware_update_runner.ex

```diff
@@ -19,6 +19,20 @@ defmodule Grizzly.FirmwareUpdates.FirmwareUpdateRunner do
 
   @default_fragment_size 1024
 
+  @doc """
+  A firmware update is in progress when a `FirmwareUpdateRunner` process is running
+  and has received at least one request for one or more image fragments.
+  """
+  @spec in_progress?() :: boolean()
+  def in_progress?() do
+    case GenServer.whereis(__MODULE__) do
+      nil -> false
+      _ -> GenServer.call(__MODULE__, :in_progress?)
+    end
```

Comment:
```suggestion
      GenServer.call(__MODULE__, :in_progress?)
```
Seems like the catch handles the error and the normal case can skip the `whereis` check.